docs/Install-with-Docker.md

```diff
@@ -58,45 +66,70 @@ docker --version
 docker-compose --version
 ```
 
-You should see a similar output:
+You should see a similar output to this;
 
 ![image](https://user-images.githubusercontent.com/75517/51273280-a132e200-19cc-11e9-914e-d54610f43ad6.png)
 
 **Note for Windows users**: you can use **git-bash** (the shell included in git) as a terminal.
 
+## PART 2 - AzerothCore
+
 ### Clone the AzerothCore repository
 
+Open a teminal such as Git Bash or Command Prompt as an Administrator.
+
 You need to clone the AzerothCore repository (or use your own fork):
 
 ```
 git clone https://github.com/azerothcore/azerothcore-wotlk.git
 ```
 
-Now go into the main directory using `cd azerothcore-wotlk`. **All commands will have to be run inside this folder**.
+Now go into the main directory using 
+
+```
+cd azerothcore-wotlk 
+```
+
+**All commands will have to be run inside this folder**.
 
 ### WoW Client Data files
 
 You also need to have the data files. Check the step "5) Download the data files" from the [installation guide](Installation#5-download-the-data-files).
 
-Put your data files into `docker/worldserver/data/` which is inside `azerothcore-wotlk`.
+Put your data files into `docker/worldserver/data/` which is inside `azerothcore-wotlk` which is likely to be found in the System32 folder if you're using Windows.
```

Comment:
o_O why is it inside `System32` ?  the `azerothcore-wotlk` is created inside the directory where you are giving:

```
git clone https://github.com/azerothcore/azerothcore-wotlk.git
```

Reply:
That's where it put it for me following these instructions. The guide at no point says where to find it, I had to find it myself, and that's where it was.

Reply:
but then we need to specify that WHEN you launch the `git clone ...` command, it will create the `azerothcore-wotlk` directory INSIDE the **CURRENT** directory, whatever your current directory is. You can check the current directory using `pwd` command on linux/mac. Maybe on git bash too.